desktop/cmp/grid/GridModel.js

```diff
@@ -163,6 +166,7 @@ export class GridModel {
         this.setSortBy(sortBy);
         this.setCompact(compact);
 
+        if (!selModel) selModel = XH.app.isMobile ? 'disabled' : 'single';
```

Comment:
use `withDefault()`, here and remove the defaulting on line 134.  This will allow applications to set to null explicitly, if they wish.